blaze-server/src/main/scala/org/http4s/server/blaze/Http1ServerStage.scala

```diff
@@ -148,8 +139,8 @@ class Http1ServerStage(service: HttpService,
 
     // Need to decide which encoder and if to close on finish
     val closeOnFinish = respConn.map(_.hasClose).orElse {
-      Connection.from(req.headers).map(checkCloseConnection(_, rr))
-    }.getOrElse(minor == 0)   // Finally, if nobody specifies, http 1.0 defaults to close
+                          Connection.from(req.headers).map(checkCloseConnection(_, rr))
```

Comment:
It seems like your editor is doing something others' are not.  I don't want to get into a style guide debate, but we all should be careful that the meaty bits of diffs aren't getting lost in whitespace changes.  (There are tools to help with this if we can't manage it ourselves.)